server/src/main/java/org/opensearch/index/remote/RemoteStoreEnums.java

```diff
@@ -167,6 +199,33 @@ long hash(PathInput pathInput) {
             }
         };
 
+        private final int code;
```

Comment:
would prefer to use `string` instead of `int` as it will be less error prone and more explicit , when we add more `PathHashAlgorithm`  .

Reply:
Using int is a standard practise since it maps elements of an enum one to one also limiting the size of the object. This has been followed in OpenSearch as well.

Reply:
Made the change.